### PR TITLE
id:86058 feat(infrastructure): initial commit

### PR DESCRIPTION
**Pull request by wenzelua**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      - deploy:
+          command: yarn run deploy
+  update-master:
+    <<: *job-defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - add_ssh_keys:
+          fingerprints:
+            - "07:8c:2b:72:8d:e7:ce:9b:8a:ed:0c:d2:e9:fe:06:6b"
```
Comment (sunstorymvp) [inline .circleci/config.yml:79]: we need to discuss this with @davesykeselateral @hhastekin-elateral

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    "import/no-useless-path-segments": "error",
+    "import/order": [ "error", {
+      "groups": [
+        [ "builtin", "external" ],
+        [ "internal" ],
+        [ "parent", "sibling", "index" ],
+      ],
+      "newlines-between": "always",
+    } ],
+    "import/prefer-default-export": "off",
```
Comment (sunstorymvp) [inline .eslintrc.js]: we can remove `import` eslint plugin and rules as not required here

File: package.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "@babel/plugin-proposal-private-methods": "^7.4.4",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.4.3",
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/is-ignored": "^7.5.1",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.8.0",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-dynamic-import-node": "^2.2.0",
+    "concurrently": "^4.1.0",
```
Comment (sunstorymvp) [inline package.json]: unused

File: package.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    "coveralls": "^3.0.3",
+    "dotenv": "^7.0.0",
+    "enzyme": "^3.9.0",
```
Comment (sunstorymvp) [inline package.json]: not required

File: webpack.optimization.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const TerserPlugin = require('terser-webpack-plugin');
+
+const minimizer = new TerserPlugin({
+  terserOptions: { parallel: true },
```
Comment (sunstorymvp) [inline webpack.optimization.js]: see web-app config

File: package.json [hunk 6]
```diff
@@ -39,8 +39,9 @@
     "coveralls": "^3.0.3",
     "dotenv": "^7.0.0",
     "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-airbnb-base": "~13.1.0",
     "eslint-import-resolver-webpack": "^0.11.1",
+    "eslint-plugin-import": "~2.17.3",
```
Comment (sunstorymvp) [inline package.json]: please use `^`, see other

**Review by sunstorymvp — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**